www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+![Combined Karmashapes input datasets](images/baralu1.png "Combined Karmashapes input datasets")
+_(above) All data sources for Baralu/Barahalu/Barahallu overlaid in QGIS_
+
+![Example of final place data for Baralu](images/baralu2.png "Example of final place data for Baralu")
+_(above) The final polygon and label position for the place of Baralu_
+
+Numerous geoprocessing tools in QGIS were used to analyze and combine these five datasets, resulting in composite polygons and points for habitations and villages in India. Each of these input datasets helped to create detailed feature attributes of Karmashapes, including:
+
+* _Population values_
+* _Population vintage_
+* _Name (Latin-based)_
+* _Name variants (Latin-based and limited local script names)_
+* _Label centroids_
+
+Collaborating with data developers and contributors to each of these projects was very helpful. The insight and knowledge they shared helped increase Karmashape’s awareness of the local administrative structure, naming conventions and regional name spellings/localizations. It took dozens of conversations with people in the [Datameet group](https://datameet.org/), colleagues from OpenStreetMap (OSM), and friendly strangers on Twitter to help answer detailed questions about rural areas and the various megacities of India.
+
+On their own, each of these datasets is busy, messy, and don’t represent locations across India well. While the PMGSY data represents habitations (the lowest level division of people loosely accounted for by the Indian government) well, the dataset lacks larger villages, towns, and cities. Karmashapes used SEDAC data to fill in most coverage gaps, along with the NGA point data. The two vectorized raster datasets then brought urban boundaries into focus and helped flush out unique location matches or agglomerations where parent-child structures are implemented to showcase the main place of the urban fabric.
+
+All of this was done through dozens of detailed matching steps, edit sessions, refining poor-quality data, cleaning/scrubbing data attributes, and squeezing out every drop of potential from each data source.
+
+![Pre-Karmashapes polygon coverage](images/pre-ks-coverage-1.png "Pre-Karmashapes polygon coverage")
+_(above) Example polygon coverage, WOF (pre-Karmashapes import)_
+
+![Post-Karmashapes polygon coverage](images/post-ks-coverage-1.png "Post-Karmashapes polygon coverage")
+_(above) Example polygon coverage, Karmashapes_
```

Comment:
To: `_(above) Example polygon coverage, raw Karmashapes_`

Reply:
(This is because some pre-Karmashapes "town" polygons are significant, but "missing" from Karmashapes, but I think the original WOF curated shape is still intact in the merged output.)